APMrover2/Log.cpp

```diff
@@ -284,11 +311,11 @@ void Rover::Log_Write_Attitude()
     DataFlash.Log_Write_Attitude(ahrs, targets);
 
 #if AP_AHRS_NAVEKF_AVAILABLE
- #if defined(OPTFLOW) and (OPTFLOW == ENABLED)
+#if defined(OPTFLOW) and (OPTFLOW == ENABLED)
```

Comment:
Yeah, I think it is bad honestly. I'm fine with space before or after the *#* but no space provides bad readability, at least for me. @magicrub can you give your input here so we can decide which way to go?